config/webspaces/example.com.xml

```diff
@@ -16,9 +16,8 @@
     </default-templates>
 
     <templates>
-        <template type="search">search.html.twig</template>
-        <template type="error">error/default.html.twig</template>
-        <template type="error-404">error/404.html.twig</template>
```

Comment:
Shouldn't we leave that one? It's a very special error page, which people probably want to override quite often.

Reply:
@danrot we never had that case. we only had in the past the case that the styling for the default error page was missing because only the 404 was styled. so I would really remove the special 404 template from the example webspace.